mscore/editstyle.cpp

```diff
@@ -679,6 +679,91 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
+      { ElementType::REST,                &EditStyle::PageMeasure  },
+      { ElementType::BAR_LINE,            &EditStyle::PageBarlines },
+      { ElementType::NOTE,                &EditStyle::PageNotes    },
+      { ElementType::CHORD,               &EditStyle::PageNotes    },
+      { ElementType::ACCIDENTAL,          &EditStyle::PageNotes    },
+      { ElementType::STEM,                &EditStyle::PageNotes    },
+      { ElementType::STEM_SLASH,          &EditStyle::PageNotes    },
+      { ElementType::LEDGER_LINE,         &EditStyle::PageNotes    },
+      { ElementType::BEAM,                &EditStyle::PageBeams    },
+      { ElementType::TUPLET,              &EditStyle::PageTuplets  },
+      { ElementType::ARPEGGIO,            &EditStyle::PageArpeggios },
+      { ElementType::SLUR,                &EditStyle::PageSlursTies },
+      { ElementType::SLUR_SEGMENT,        &EditStyle::PageSlursTies },
+      { ElementType::TIE,                 &EditStyle::PageSlursTies },
+      { ElementType::TIE_SEGMENT,         &EditStyle::PageSlursTies },
+      { ElementType::HAIRPIN,             &EditStyle::PageHairpins },
+      { ElementType::HAIRPIN_SEGMENT,     &EditStyle::PageHairpins },
+      { ElementType::VOLTA,               &EditStyle::PageVolta    },
+      { ElementType::VOLTA_SEGMENT,       &EditStyle::PageVolta    },
+      { ElementType::OTTAVA,              &EditStyle::PageOttava   },
+      { ElementType::OTTAVA_SEGMENT,      &EditStyle::PageOttava   },
+      { ElementType::PEDAL,               &EditStyle::PagePedal    },
+      { ElementType::PEDAL_SEGMENT,       &EditStyle::PagePedal    },
+      { ElementType::TRILL,               &EditStyle::PageTrill    },
+      { ElementType::TRILL_SEGMENT,       &EditStyle::PageTrill    },
+      { ElementType::VIBRATO,             &EditStyle::PageVibrato  },
+      { ElementType::VIBRATO_SEGMENT,     &EditStyle::PageVibrato  },
+      { ElementType::BEND,                &EditStyle::PageBend     },
+      { ElementType::TEXTLINE,            &EditStyle::PageTextLine },
+      { ElementType::TEXTLINE_SEGMENT,    &EditStyle::PageTextLine },
+      { ElementType::ARTICULATION,        &EditStyle::PageArticulationsOrnaments },
+      { ElementType::FERMATA,             &EditStyle::PageFermatas },
+      { ElementType::STAFF_TEXT,          &EditStyle::PageStaffText },
+      { ElementType::TEMPO_TEXT,          &EditStyle::PageTempoText },
+      { ElementType::LYRICS,              &EditStyle::PageLyrics   },
+      { ElementType::LYRICSLINE,          &EditStyle::PageLyrics   },
+      { ElementType::LYRICSLINE_SEGMENT,  &EditStyle::PageLyrics   },
+      { ElementType::DYNAMIC,             &EditStyle::PageDynamics },
+      { ElementType::REHEARSAL_MARK,      &EditStyle::PageRehearsalMarks },
+      { ElementType::FIGURED_BASS,        &EditStyle::PageFiguredBass },
+      { ElementType::HARMONY,             &EditStyle::PageChordSymbols },
```

Comment:
Appending `{ ElementTypr::FRET_DIAGRAM,        &EditStyle::PageFretboardDiagrams }` enables the "Style..." entry on Fretboard Diagrams. Didn't you add them intentionally?

Reply:
Yep, that's what I've just added and pushed. Should be fixed now. I think it was just an oversight on my part :)